app/src/main/AndroidManifest.xml

```diff
@@ -52,6 +52,9 @@
     <activity
       android:name=".profile.ProfileActivity"
       android:theme="@style/OppiaThemeWithoutActionBar" />
+    <activity
+      android:name=".settings.administrator.AdministratorControlsActivity"
```

Comment:
Actually the package name for this should be `administratorcontrols` and also, it should be directly inside `app` not inside `settings`.

Reply:
Done.